cmd/process-agent/collector_test.go

```diff
@@ -111,32 +113,29 @@ func TestDisableRealTime(t *testing.T) {
 		{
 			name:            "true",
 			disableRealtime: true,
-			expectedChecks:  []checks.Check{checks.Process, checks.Container},
+			expectedChecks:  []checks.Check{checks.Container},
```

Comment:
Would you mind clarifying why we shouldn't expect `process` check here?

Reply:
Since we're moving the responsibility of disabling RT checks onto `getChecks`, we need start taking into account that the `Process` and `Container` checks are mutually exclusive and cannot be enabled at the same time. 

The reason I chose `Container` checks is because the regular `Container` check and the `RTContainer` check are different, so I thought it would be easier to test. Looking at it now, I think I will add add another test for `Process` checks too, since enabling or disabling rt checks for that is up to the collector

Reply:
Thanks for clarifying it! Adding a test case for `Process` looks a great idea!